msal/build.gradle

```diff
@@ -145,9 +145,9 @@ dependencies {
         transitive = false
     }
 
-    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '2.0.3-SNAPSHOT', changing: true)
+    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '2.0.6-SNAPSHOT', changing: true)
 
-    distApi("com.microsoft.identity:common:2.0.2") {
+    distApi("com.microsoft.identity:common:2.0.2-hf1") {
```

Comment:
should this be 2.0.6?

Reply:
Yes, but this is being merged into dev no?

Reply:
Yes, this is being merged into dev

Reply:
dist API version of MSAL in `dev` is `1.4.1` which depends on `2.0.2-hf1` , `2.0.6` was for `broker`